Tests/test_deprecate.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from PIL import _deprecate
+
+
+@pytest.mark.parametrize(
+    "version, expected",
+    [
+        (
+            10,
+            r"Old thing is deprecated and will be removed in Pillow 10 "
```

Comment:
```suggestion
            "Old thing is deprecated and will be removed in Pillow 10 "
```
Technically, `r` isn't needed here, since there are no backslashes in this line. But I'm guessing you'd rather play it safe for a theoretical future where this line and the next shift around?

Reply:
I'd meant to leave it out, lining should catch if it's missing in the future. Thanks!